### ISSUE TITLE
Error when using VertexAI Gemini 1.5 Pro model for conversation

### ISSUE THREAD

**Issue by shelltea** [langgenius/dify#4981]
### Self Checks

- [X] This is only for bug report, if you would like to ask a question, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).
- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] Please do not modify this template :) and fill in all the required fields.

### Dify version

0.6.10

### Cloud or Self Hosted

Self Hosted (Docker)

### Steps to reproduce

1. Use Dify version 0.6.10.
2. Select VertexAI Gemini 1.5 Pro model for conversation.
3. Observe the error message.

![Xnip2024-06-06_08-33-26](https://github.com/langgenius/dify/assets/864375/7d85b42e-eed8-482a-9cdd-b1e7bdb5c6e1)

### ✔️ Expected Behavior

The VertexAI Gemini 1.5 Pro model should work correctly for conversation.

### ❌ Actual Behavior

An error occurs:

```
[vertex_ai] Error: module 'vertexai.generative_models' has no attribute 'Schema'
```

**Comment by crazywoola** [comment]
Not sure if this is fixed in #4971 @hellof20 Could you take a look at this as well if you have some spare time?

**Comment by hellof20** [comment]
@crazywoola I can reproduce this problem. When the user uses the Gemini model in the agent and includes tool, the error will occur. I need some time to figure out how to fix this.
#4971 won't solve this problem.
